projects/plugins/jetpack/changelog/add-mailchimp-block-tests

```diff
@@ -0,0 +1,3 @@
+Significance: patch
```

Comment:
Looks like this filename is `add-mailchimp-block-tests` but should be `add-map-block-tests` 🙂

Reply:
doh, the dangers of copy pasta! - fixed